### PR TITLE
Small bug fix

### PR DESCRIPTION
**Pull request by james-ben**
making it so the RPi server correctly serves up static javascript files.

### CODE REVIEW

**Comment by james-ben** [comment]
Also updated the row and column spec for how the lights are arrayed this year.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Neopixel Christmas Lights
```
Comment (brinbh) [inline README.md:1]: Maybe we should add a part about how to change the row size on the tree for others to use.

File: code/light_utils/fake_grid.py [hunk 2]
```diff
@@ -36,38 +40,27 @@ def __del__(self):
...
+
+		# rows
+		pts = [0, 21, 34, 46, 57, 66, 76, 85, 94, 100]
+		gen = itertools.tee(pts)
+		next(gen[1])
+		# https://stackoverflow.com/a/5434936/12940429
+		self.rows = [(next(gen[0]), next(gen[1])) for _ in range(len(pts)-1)]
+
+		# init the columns procedurally
+		self.num_cols = 6
```
Comment (brinbh) [inline code/light_utils/fake_grid.py:52]: this is a much better way to do this.

File: code/networking/flask-server.py [hunk 3]
```diff
@@ -47,13 +47,15 @@ def __init__(self):
 		self.strobe = strobe.StrobeLights()
 		self.column = columns.ColumnLights()
 		self.blinker = blink.BlinkLights()
+		self.crazy = crazy.CrazyOldColumnLights()
```
Comment (brinbh) [inline code/networking/flask-server.py:50]: can you just call these `crazyLights`? or maybe `wackoLights`

File: code/networking/flask-server.py [hunk 4]
```diff
@@ -47,13 +47,15 @@ def __init__(self):
...
 		self.blinker = blink.BlinkLights()
+		self.crazy = crazy.CrazyOldColumnLights()
 
 		self.functionMap = {
 			"twinkle": self.twinkler.run,
 			"stripes": self.striper.run,
 			"strobe": self.strobe.run,
 			"columns": self.column.run,
 			"blink": self.blinker.run,
+			"crazy": self.crazy.run,
```
Comment (brinbh) [inline code/networking/flask-server.py:58]: oh I see you just call them cazy here. As long as the `OldColumn` part isn't everywhere.

**Review by brinbh — APPROVED** [review]

**Pull request merged**